examples.md

```diff
@@ -157,3 +158,29 @@ Using [NuGet lock files](https://docs.microsoft.com/nuget/consume-packages/packa
     restore-keys: |
       ${{ runner.os }}-pods-
 ```
+
+## Docker
+
+```yaml
+- uses: actions/cache@v1
+  id: cache
+  with:
+    path: docker-cache
+    key: ${{ runner.os }}-docker-${{ github.sha }}
+    restore-keys: |
+      ${{ runner.os }}-docker-
+- name: Load cached Docker layers
+  run: |
+    if [ -d "docker-cache" ]; then
+      cat docker-cache/x* > my-image.tar
+      docker load < my-image.tar
+      rm -fr docker-cache
+    fi
+- name: Build image
+  if: steps.cache.outputs.cache-hit != 'true'
+  run: |
+    docker build --cache-from my-image -t my-image .
+    docker save my-image $(docker history -q my-image | awk '!/<missing>/{print}') > my-image.tar
+    mkdir docker-cache
+    split -b 100m my-image.tar docker-cache/x
```

Comment:
Had a question about the splitting strategy - wouldn't the entire `docker-cache` directory be cached as an individual cache item? 

If that's the case, and if the size of the `docker-cache` directory is over 400mb, then splitting the tar wouldn't help much unless the split files are put into separate folders that are cached using different keys.

Reply:
I don't think it's necessary to have cache hits because when falling back to cache restore it loads the layers into Docker to be used in its own `--cache--from` mechanism. That's why this example caches the whole directory against the commit SHA.

Reply:
docker-cache (total of over 400mb)
└xaa (100mb)
└xab (100mb)
└xac (100mb)
└xad (100mb)
└xae (100mb)

Reply:
@sousousore1 thanks for explaining. That was the point I was trying to make in my comment. 

I ended up creating a new folder for each file I was going to cache using some bash helper functions

```
folder-a (400mb)
 └ xaa
folder-b (400mb)
 └ xab
folder-c (400mb)
 └ xac
```

After restoring cache, I combined them into one file `cat folder-*/* > docker-cache`

Reply:
Apologies, I misunderstood the original question. Thanks for explaining. I'll try and reproduce this.

Reply:
Cache size was increase to 2GB #140

Reply:
Cache size was increased to 5GB

Reply:
Cache size was increased to `10GB`.